helperFunctions/subscribeHelpers/InitiateSalesforceSessionHelpers/CheckChatStatusHelper.ts

```diff
@@ -145,6 +147,16 @@ export class CheckChatStatus {
 							}
 							await checkAgentStatusDirectCallback.checkAgentStatusCallbackError(this.technicalDifficultyMessage);
 							return;
+						} else if (isAgentDisconnected === true) {
+							console.log(ErrorLogs.AGENT_DISCONNECTED);
+							this.modify
+								.getAnalytics()
+								.sendEvent(getEventData(this.data.room.id, EventName.AGENT_TRANSFER_FAILED_DUE_TO_DISCONNECTED_AGENT));
+							await checkAgentStatusDirectCallback.checkAgentStatusCallbackError(
+								CloseMessages.CHAT_CLOSED_DUE_TO_DISCONNECTED_AGENT,
+								EventName.AGENT_TRANSFER_FAILED_DUE_TO_DISCONNECTED_AGENT,
+							);
```

Comment:
This will close the chat, right? As @ear-dev mentioned we are receiving this message when the salesforce agent transfers chat to another salesforce agent. So it will just close the chat in between.